Vagrantfile

```diff
@@ -0,0 +1,41 @@
+# -*- mode: ruby -*-
```

Comment:
I'm ignoring your module 4 changes for now as this branch was supposed to be for modules 2/3